ios/app/Settings.bundle/Root.plist

```diff
@@ -18,7 +18,7 @@
 			<key>Key</key>
 			<string>MGLMapboxMetricsEnabled</string>
 			<key>Title</key>
-			<string>Mapbox Metrics</string>
+			<string>Mapbox Telemetry</string>
```

Comment:
Also update the corresponding [.strings files](https://github.com/mapbox/mapbox-gl-native/blob/master/ios/app/Settings.bundle/en.lproj/Root.strings).